openapi.json

```diff
@@ -303,16 +321,47 @@
                             }
                           }
                         },
-                        "PNG": {
+                        "GeoPackage": {
+                          "gis_data_types": [
+                            "raster",
+                            "vector"
+                          ],
                           "parameters": {
-                            "worldfile": {
-                              "type": "boolean",
-                              "description": "Force the generation of an associated ESRI world file.",
-                              "default": false
+                            "gis_data_type": {
```

Comment:
I'm not sure if this schema definition under `parameters` is intenionally or not. Probably this needs to be removed.

Reply:
That's intentional, as discussed.